samples/client/petstore/java-micronaut-client/src/test/groovy/org/openapitools/api/PetApiSpec.groovy

```diff
@@ -74,6 +75,7 @@ class PetApiSpec extends Specification {
      * Finds Pets by status
      * Multiple status values can be provided with comma separated strings
      */
+    @Ignore("Issue reported in https://github.com/micronaut-projects/micronaut-jackson-xml/issues/175")
```

Comment:
Note: This and the next test failed because of the issue reported in https://github.com/micronaut-projects/micronaut-jackson-xml/issues/175:
`phototoUrls` is an empty set, but is deserialized as set with an empty String (`[""]`). No changes are required on the openapi generator side.